src/installation/guides/chroot.md

```diff
@@ -0,0 +1,316 @@
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+This step is separated into the XBPS method and the ROOTFS method. Follow only
+one of the two following subsections.
+
+### The XBPS Method
+
+[Select an XBPS mirror.](../../xbps/repositories/mirrors/index.md) For
+simplicity, save this URL to a shell variable for later use. Append `/current`
+to this repository url to install a glibc distribution, or `/current/musl` for a
+musl distribution.
+
+> Note: musl binaries are not available for i686.
+
+For example, this is the url for glibc from the Tier 1 server in Finland,
+
+```
+# REPO=https://alpha.de.repo.voidlinux.org/current
+```
+
+XBPS also needs to know what architecture is being installed, available options
+are `x86_64`, `x86_64-musl` and `i686` for PC architecture computers.
+
+For example,
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Use [xbps-install](https://man.voidlinux.org/xbps-install.1) to boot strap the
+installation by installing the `base-system` metapackage.
+
+```
+# XBPS_ARCH=$ARCH xbps-install -S -r /mnt -R "$REPO" base-system
+```
+
+### The ROOTFS Method
+
+[Download a ROOTFS
+tarball](https://voidlinux.org/download/#download-installable-base-live-images-and-rootfs-tarballs)
+matching your architecture.
+
+Unpack the tarball into the newly configured filesystems.
+
+```
+# tar xvf void-[...]-ROOTFS.tar.xz -C /mnt
+```
+
+## Configuration
+
+> Note: With the exception of the section "Install base-system (ROOTFS method
+> only) the remainder of this guide is common to both XBPS and ROOTFS
+> installation methods.
+
+### Entering the Chroot
+
+Mount the pseudo-filesystems needed for a chroot,
+
+```
+# mount --rbind /sys /mnt/sys && mount --make-rslave /mnt/sys
+# mount --rbind /dev /mnt/dev && mount --make-rslave /mnt/dev
+# mount --rbind /proc /mnt/proc && mount --make-rslave /mnt/proc
+```
+
+Copy the DNS configuration into the new root so that XBPS can still download new
+packages inside the chroot,
+
+```
+# cp /etc/resolv.conf /mnt/etc/
+```
+
+Chroot into the new installation
+
+```
+# PS1='(chroot) # ' chroot /mnt/ /bin/bash
+```
+
+### Install base-system (ROOTFS method only)
+
+> Note: This section can be skipped if following the XBPS method. You already
+> have base-system installed.
+
+ROOTFS images generally contain out of date software, due to being a snapshot of
+the time when they were built, and do not come with a complete `base-system`.
+Update the package manager and install `base-system`.
+
+```
+# xbps-install -u xbps
+# xbps-install base-system
+# xbps-remove base-voidstrap
+```
+
+### Distribution Configuration
+
+Write the hostname to `/etc/hostname`. Go through the options in
```

Comment:
Installation instead of distribution? IDK

Reply:
sure